contribute.md

```diff
@@ -0,0 +1,77 @@
+# Contribute to Human-Connection
```

Comment:
I believe Github has a naming convention here, this file ought to be called `CONTRIBUTING.md`